### PR TITLE
Add/careers page

### PR DESCRIPTION
**Pull request by Soujanya2107**
Career page with footer changes on main page and modal .
I have added the job postings as well.
Please review this and let me know if anything else has to be done before merging to staging.

### CODE REVIEW

File: components/navbar.js [hunk 1]
```diff
@@ -72,11 +72,11 @@ class NavLinks extends React.Component {
           <a> FAQ's</a>
         </Link>
       </Menu.Item>
-      {/* <Menu.Item key="2">
+       <Menu.Item key="2">
```
Comment (sonu-sonam) [inline components/navbar.js:75]: double "key=2" for both careers and contact-us,
also do not show in production, look blog menu for reference.

File: pages/careers.js [hunk 2]
```diff
@@ -45,19 +45,31 @@ class Careers extends React.Component {
   }
 
   state = {
-    applyJobModalVisible: false
+    applyJobModalVisible: false,
+    applyJob2ModalVisible: false
```
Comment (sonu-sonam) [inline pages/careers.js:49]: better to use good naming convention

File: pages/careers.js [hunk 3]
```diff
@@ -45,19 +45,31 @@ class Careers extends React.Component {
...
   }
+
+  applyJob2ModalVisible(applyJob2ModalVisible) {
+    this.setState({ applyJob2ModalVisible });
+  }
+
   handleOk = () => {
     this.setState({ applyJobModalVisible: false });
   };
+  handleOk = () => {
```
Comment (sonu-sonam) [inline pages/careers.js:63]: two handleOk functions?

File: pages/careers.js [hunk 4]
```diff
@@ -45,19 +45,31 @@ class Careers extends React.Component {
...
+    this.setState({ applyJob2ModalVisible: false });
+  };
 
   handleCancel = () => {
     this.setState({ applyJobModalVisible: false });
   };
+  handleCancel = () => {
```
Comment (sonu-sonam) [inline pages/careers.js:70]: remove two same functions with same name.

File: pages/careers.js [hunk 5]
```diff
@@ -345,189 +333,19 @@ class Careers extends React.Component {
           <Modal
             className="apply-job-modal"
             visible={this.state.applyJobModalVisible}
-            onOk={() => this.applyJobModalVisible(false)}
-            onCancel={() => this.applyJobModalVisible(false)}
-            footer={[
-              <div className="apply-job">Apply for this position:</div>,
-              <Formik
```
Comment (sonu-sonam) [inline pages/careers.js:352]: Why are we removing the job apply form? Did Charlie ask for this?

File: pages/careers.js [hunk 6]
```diff
@@ -542,54 +360,101 @@ class Careers extends React.Component {
...
-                  imperdiet.
+                  Familiarity using Scrum/Agile development methodologies
                 </li>
+                <li>Experience building object oriented web applications in JavaScript, HTML5, CSS and React</li>
               </ul>
             </div>
+
           </Modal>
+          
+          <Modal
```
Comment (sonu-sonam) [inline pages/careers.js:393]: The code is getting repeated. We should follow DRY (Don't Repeat Yourself) principle.

File: pages/careers.js (hunk 5 above)
Comment (Soujanya2107) [reply pages/careers.js:352]: Yes Charlie had asked to remove that complete form and a message to drop a email to careers@relaypay.io instead.

File: components/navbar.js (hunk 1 above)
Comment (Soujanya2107) [reply components/navbar.js:76]: This is fixed, Please have a look.

File: pages/careers.js (hunk 3 above)
Comment (Soujanya2107) [reply pages/careers.js:63]: Fixed this.

File: pages/careers.js (hunk 4 above)
Comment (Soujanya2107) [reply pages/careers.js:70]: Fixed this.

File: pages/careers.js (hunk 2 above)
Comment (Soujanya2107) [reply pages/careers.js:49]: Can you suggest of something?

**Pull request closed without merging**